### PR TITLE
added a personal mark

### PR DESCRIPTION
**Pull request by Cuaima**
Here is a change in the repository

### CODE REVIEW

**Comment by Cuaima** [comment]
He Mislav, it was great meeting you!

File: README.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ Is Rotterdam a must-see city? Absolutely!
 Rotterdam has a rich cultural scene centred largely around the cultural axis between Museumpark and Witte de Withstraat. The area includes acclaimed museums like Museum Boijmans van Beuningen, The New Institute and the Kunsthal, presenting a widely varied range of art exhibitions.
 
 TENT and Witte de With guarantee a relevant, innovative, experimental programme, and the numerous galleries around Witte de Withstraat and outside the city centre are also well worth your time. 
+
+The Claw was here!
```
Comment (mislav) [inline README.md]: That's great!

…
…
…
who is the Claw again?

**Comment by fadikurdi** [comment]
j1sus

File: README.md [hunk 2]
```diff
@@ -10,3 +10,5 @@ Is Rotterdam a must-see city? Absolutely!
 Rotterdam has a rich cultural scene centred largely around the cultural axis between Museumpark and Witte de Withstraat. The area includes acclaimed museums like Museum Boijmans van Beuningen, The New Institute and the Kunsthal, presenting a widely varied range of art exhibitions.
 
 TENT and Witte de With guarantee a relevant, innovative, experimental programme, and the numerous galleries around Witte de Withstraat and outside the city centre are also well worth your time. 
+
```
Comment (shahrokhnabavi) [inline README.md]: Why have we an empty line here?

**Pull request merged**